DQM/SiStripMonitorClient/plugins/SiStripAnalyser.cc

```diff
@@ -1,37 +1,92 @@
-/*
- * \file SiStripAnalyser.cc
- *
- * \author  S. Dutta INFN-Pisa
- *
- */
-
-#include "SiStripAnalyser.h"
-
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/LuminosityBlock.h"
-#include "FWCore/MessageLogger/interface/MessageLogger.h"
-#include "FWCore/ParameterSet/interface/FileInPath.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "FWCore/ServiceRegistry/interface/Service.h"
-#include "FWCore/Utilities/interface/InputTag.h"
-
-#include "DQMServices/Core/interface/DQMStore.h"
 
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CondFormats/DataRecord/interface/SiStripCondDataRecords.h"
 #include "DataFormats/FEDRawData/interface/FEDNumbering.h"
 #include "DataFormats/FEDRawData/interface/FEDRawData.h"
 #include "DataFormats/FEDRawData/interface/FEDRawDataCollection.h"
 #include "DataFormats/SiStripDetId/interface/StripSubdetector.h"
-
 #include "DQM/SiStripCommon/interface/SiStripFolderOrganizer.h"
+#include "DQM/SiStripMonitorClient/interface/SiStripActionExecutor.h"
 #include "DQM/SiStripMonitorClient/interface/SiStripUtility.h"
+#include "DQM/SiStripMonitorSummary/interface/SiStripClassToMonitorCondData.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
 
 #include <cmath>
 #include <cstdio>
 #include <iomanip>
 #include <iostream>
+#include <map>
+#include <vector>
 #include <sstream>
 #include <string>
 
+class SiStripWebInterface;
+class SiStripFedCabling;
+class SiStripDetCabling;
+class SiStripClassToMonitorCondData;
+class FEDRawDataCollection;
+
+class SiStripAnalyser : public edm::one::EDAnalyzer<> {
```

Comment:
Is this the right fix?
I think this should be moved to `DQMEDHarvester` instead.